src/services/transactionService.ts

```diff
@@ -176,9 +176,15 @@ async function getTransactionDetailForNormalTransfer(
   const transaction = await getNetworkWeb3(networkId).eth.getTransaction(
     txHash,
   );
+  const receipt = await getNetworkWeb3(networkId).eth.getTransactionReceipt(
+    txHash,
+  );
```

Comment:
@CarlosQ96 
```
getNetworkWeb3(networkId).eth.getTransaction(
    txHash,
  );
  ```
  return the tx info and it doesn't matter tx is failed or not, so we have to use
  ```
   getNetworkWeb3(networkId).eth.getTransactionReceipt(
    txHash,
  );
  ```
  for this purpose